src/dubois.cc

```diff
@@ -0,0 +1,37 @@
+#include "dubois.h"
+
+bool change_color = false;
```

Comment:
I think that might cause issues with multiple windows as this would switch globally for the whole application. Maybe the code could be converted to a class which is instantiated for each main window?

Reply:
Or this code could be added to Renderer as a function?